lib/src/flutter_downloader.dart

```diff
@@ -0,0 +1,136 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:crypto/crypto.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_downloader/flutter_downloader.dart';
+import 'package:flutter_downloader/src/dart_download.dart';
+import 'package:flutter_downloader/src/platform_download.dart';
+
+part 'legacy_api.dart';
```

Comment:
I'd deprecate this in v2.0 so we can just focus on supporting a single way of doing this.